### ISSUE TITLE
JSON.parse needs to be quoted

### ISSUE THREAD

**Issue by diminator** [oceanprotocol/secret-store-client-js#5]
## Expected Behavior

encrypt string with python
decrypt string with JS
plaintext is equal

## Current Behavior

errors on preparing the result (but rightfully decrypts)

## Failure Information (for bugs)

```
SyntaxError: Unexpected token a in JSON at position 0
    at JSON.parse (<anonymous>)
    at /home/dimi/oceanprotocol/tuna/node_modules/@oceanprotocol/secret-store-client/dist/ParityClient.js:85:25
```
### Possible Solution

double quote the `documentString` (if it wasn't double quoted) before calling `JSON.parse`

## Specifications

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Version: x.x.x
* Python: 3.6
* Solidity: x.x.x
* JavaScript: x.x.x
* Node: x.x.x
* OS: linux

## Failure Logs

Please include any relevant log snippets or files here.

**Comment by H34D** [comment]
Can you supply an example please? Cannot reproduce.

**Comment by H34D** [comment]
We forgot to implement error handling in the parity client. I added this.

**Comment by H34D** [comment]
fixed with #8
